### DISCUSSION TITLE
Необходимость новой лицензии

### DISCUSSION THREAD

**Discussion by ivansglazunov** [deep-foundation/dev#37] [Posts]
Нужно разработать лицензию отдающую в общественное достояние результаты работы и утверждающую что организация/фонд **берет на себя ответственность** за то что ресурсы фонда при их наличии и соответствии уставным приоритетам занимается развитием и поддержкой того что есть, но не несёт ответственности за использование, и за то к каким последствиям привело.

Авторы не авторское право.

Должен быть список contributors но не авторов. Те кто сделали вклад в создание, да. Авторское право - за человечеством. Нет владельца и автора, но есть те кто несут ответственность. Перечисление контрибьюторов и ссылки на фонд является обязательным критерием новой лицензии.

> Ваши безответственные лицензии и стремление к доходу и авторству привели мир к такому состоянию. Языи и модели не совместимы, холивары и личные эго цели в приоритете над консеквенциолизмом. Нам не нужно авторство, нам не нужен доход последвтий. Нам нужны сами последствия. Мы берем ответственность за то что эти последствия делаются, будут делаться, поддерживаться и улучшаться пока обществу нужно наше существования. Мы гарантируем что наши ресурсы будут распределяться на поддержание общественного достояния взятого на поддержание фондом и гарантируем это нашим существованием и ресурсами вкладываемыми в фонд.

**Comment by zakhar-kogan** [comment]
По ощущению от прочтения текста внизу - ту мач [Грета Тунберг](https://user-images.githubusercontent.com/36503576/121936476-57dc4d80-cd52-11eb-9793-180197433deb.png)
Хотя мб так и лучше 🤔

Разве нельзя в обычную лицензию указать список контрибьюторов?
https://github.com/deepcase/deepcase/graphs/contributors => https://opensource.stackexchange.com/questions/8616/how-to-note-contributors-work-with-mit-license наподобие такого, но без копирайта
Ну, или модифицировать нужную лицензию под это; кажется, пару пунктов достаточно будет добавить; наподобие 
_1. Contributors are the ones investing their time and skills into creating the Product and are mentioned in the License as well as Deep Foundation.
2. Under no circumstances are Contributors deemed to be responsible for any consequences arising from the Product's use by any third party.
3. There is no licensor, the Product is distributed as a public commons.
4. Deep Foundation takes responsibility to support and manage further development of the Product to the best of its resources, but cannot guarantee a strict development roadmap or a timeline, and cannot be responsible for any consequences or damage arising from the Product or any of its parts being not up-to-date to perform any operation demanded or implied._
